src/Mod/Sketcher/App/PythonConverter.cpp

```diff
@@ -312,18 +323,27 @@ PythonConverter::SingleGeometry PythonConverter::process(const Part::Geometry* g
                  for (auto& pole : poles) {
                      stream << "App.Vector(" << pole.x << "," << pole.y << "),";
                  }
-                 std::string controlpoints = stream.str();
-                 // remove last comma and add brackets
-                 int index = controlpoints.rfind(',');
-                 controlpoints.resize(index);
-                 controlpoints.insert(0, 1, '[');
-                 controlpoints.append(1, ']');
+                 std::string controlpoints = makeArryString(stream);
+
+                 stream = std::stringstream();
+                 std::vector<int> mults = bSpline->getMultiplicities();
+                 for (const auto& mult : mults) {
+                     stream << mult << ",";

```

Comment:
```suggestion
                     stream << mult << ", ";

```

This should work with the `int index = res.rfind(',');` lines above (which I ironically suggested are not necessary). Would be good to have a well formatted input in python as well, though perhaps it should be done automatically as an intermediate step.